packages/riipen-ui/src/components/Chip.test.jsx

```diff
@@ -0,0 +1,136 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Chip from "./Chip";
+
+it("renders without errors", () => {
+  try {
+    mount(<Chip />);
+  } catch (error) {
+    expect(true).toEqual(false);
+  }
+});
+
+describe("default props", () => {
+  it("sets correct default props", () => {
+    const defaultProps = new Chip().type.defaultProps;
+    defaultProps.classes.push("riipen-chip", "riipen");
+
+    const wrapper = mount(<Chip />);
+
+    const component = wrapper.find("Chip").childAt(0);
+    expect(component.name()).toEqual("div");
+    expect(component.hasClass(defaultProps.color)).toEqual(true);
+    expect(component.hasClass("disabled")).toEqual(false);
+    expect(component.hasClass("hover")).toEqual(false);
+    expect(component.hasClass(defaultProps.size)).toEqual(true);
+    expect(component.hasClass(defaultProps.variant)).toEqual(true);
+    expect(component.childAt(0).name()).toEqual("span");
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+});
+
+describe("variant prop", () => {
+  it("outlines chip given outlined variant", () => {
+    const wrapper = mount(<Chip variant="outlined" />);
+
+    expect(
+      wrapper
+        .find("Chip")
+        .childAt(0)
+        .hasClass("outlined")
+    ).toEqual(true);
+  });
+});
+
+describe("component prop", () => {
+  it("component can be any element", () => {
```

Comment:
For the name of this test, again I would try to be as consistant as possible:

1. Does it work given a good value
2. Does it work given a bad value

So I would maybe try to align this with the wording for the color and variant prop as previously described. Maybe something like `"uses custom component"`.